src/main/java/util/FileHelper.java

```diff
@@ -0,0 +1,109 @@
+package util;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ * A helper that manages file I/O.
+ */
+public class FileHelper {
+    private static final Logger logger = LogManager.getLogger(FileHelper.class.getName());
+
+    private static final String CHARSET = "UTF-8";
+    private static final String EMPTY_STRING = "";
+
+    private final String configFileDirectory;
+    private final String configFileName;
+
+    /**
+     * Initialise the FileHelper with the directory and the file name
+     * @param configFileDirectory the directory that the config file is located in
+     * @param configFileName the name of the config file
+     */
+    public FileHelper(String configFileDirectory, String configFileName) {
+        this.configFileName = configFileName;
+        this.configFileDirectory = configFileDirectory;
+    }
+
+    /**
+     * Returns the File representation at the config file path
+     */
+    private File getFile() {
+        createDirectoryIfNotExist();
+        return new File(configFileDirectory, configFileName).getAbsoluteFile();
+    }
+
+    /**
+     * Attempts to create the config directory if it does not exist.
+     */
+    private void createDirectoryIfNotExist() {
+        File directory = new File(configFileDirectory);
+        boolean directoryExists = directory.exists() && directory.isDirectory();
+        if (!directoryExists) {
+            if (directory.mkdirs()) {
+                logger.info("Config directory created: " + directory.toString());
+            } else {
+                logger.warn("Could not create config file directory");
```

Comment:
If it can't be handled here, an exception should be thrown so the caller can handle it.

If we just log, then continue as if we were able to create the directory, code which comes after may run into problems.